doc/classes/AStarGrid2D.xml

```diff
@@ -12,105 +23,156 @@
+				[b]Note:[/b] This method is not thread-safe. If called from a [Thread], it will return an empty [PackedVector3Array] and will print an error message.
 			</description>
 		</method>
 		<method name="is_dirty" qualifiers="const">
 			<return type="bool" />
 			<description>
+				Indicates that the grid parameters was changed and the [method update] needs to be called.
 			</description>
 		</method>
 		<method name="is_in_bounds" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="x" type="int" />
 			<param index="1" name="y" type="int" />
 			<description>
+				Returns [code]true[/code] if the [param x] and [param y] is a valid grid coordinate (id).
 			</description>
 		</method>
 		<method name="is_in_boundsv" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="id" type="Vector2i" />
 			<description>
+				Returns [code]true[/code] if the [param id] vector is a valid grid coordinate.
 			</description>
 		</method>
 		<method name="is_point_solid" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="id" type="Vector2i" />
 			<description>
+				Returns whether a point is disabled or not for pathfinding. By default, all points are enabled.
 			</description>
 		</method>
 		<method name="set_point_solid">
 			<return type="void" />
 			<param index="0" name="id" type="Vector2i" />
 			<param index="1" name="solid" type="bool" default="true" />
 			<description>
+				Disables or enables the specified point for pathfinding. Useful for making a temporary obstacle.
 			</description>
 		</method>
 		<method name="update">
 			<return type="void" />
 			<description>
+				Updates the internal state of the grid to prepare it to search the path.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="cell_size" type="Vector2" setter="set_cell_size" getter="get_cell_size" default="Vector2(1, 1)">
+			The size of the point cell which will be applied to calculate the resulted point position returned by the [method get_point_path]. After changed, the [method update] needs to be called before using the [AStarGrid2D] to calculate path.
 		</member>
 		<member name="default_heuristic" type="int" setter="set_default_heuristic" getter="get_default_heuristic" enum="AStarGrid2D.Heuristic" default="0">
+			A default [enum Heuristic] which will be used to calculate the path if the [method _compute_cost] and/or [method _estimate_cost] was not overrided.
 		</member>
 		<member name="diagonal_mode" type="int" setter="set_diagonal_mode" getter="get_diagonal_mode" enum="AStarGrid2D.DiagonalMode" default="0">
+			A specific [enum DiagonalMode] mode which will force the path to avoid or accept the specific diagonals.
 		</member>
 		<member name="jumping_enabled" type="bool" setter="set_jumping_enabled" getter="is_jumping_enabled" default="false">
+			Enabled or disables the jumping to skip up the intermediate points and speed up the searching algorithm.
```

Comment:
```suggestion
			Enables or disables jumping to skip up the intermediate points and speeds up the searching algorithm.
```
Maybe "up _to_ the intermediate points", but I'm not sure what those are exactly, maybe this one needs more explanation actually :)